react/features/filmstrip/middleware.web.js

```diff
@@ -53,6 +57,21 @@ MiddlewareRegistry.register(store => next => action => {
             store.dispatch(setVerticalViewDimensions());
             break;
         }
+
+        if (isFilmstripResizable(state)) {
+            const { width: filmstripWidth } = state['features/filmstrip'];
+            const { clientWidth } = action;
+            let width;
+
+            if (filmstripWidth.current > clientWidth - MIN_STAGE_VIEW_WIDTH) {
+                width = Math.max(clientWidth - MIN_STAGE_VIEW_WIDTH, DEFAULT_FILMSTRIP_WIDTH);
+            } else {
+                width = Math.min(clientWidth - MIN_STAGE_VIEW_WIDTH, filmstripWidth.userSet);
+            }
+
+
+            store.dispatch(setFilmstripWidth(width));
```

Comment:
Do you think it’s worth it to add a check here and dispatch the action only if we change the value?

I'd imagine that most of the times we'll just dispatch the same value, isn't it?